### ISSUE TITLE
Translated README in Bahasa

### ISSUE THREAD

**Issue by zulkiflizaki** [huseinzol05/malaya#95]
Ada cadangan untuk terjemah keseluruhan projek ke dalam Bahasa Melayu? Memandangkan ini adalah projek _NLP_ Bahasa, mungkin lagi senang _engagement_ kalau ada versi Bahasa Melayu. Sila maklum jika perlu bantuan.

**Comment by huseinzol05** [comment]
Saya kena maintain 2 folders seperti yang dicadangkan oleh readthedocs, https://docs.readthedocs.io/en/stable/localization.html.
Terjemah docs/*.ipynb dan docs/*.rst, banyak juga

**Comment by zulkiflizaki** [comment]
Betul. Saya dah tengok fail dalam folder, agak banyak juga. Berkemungkinan akan ada masalah untuk terjemahan sebab _library name_ guna _English by default_.

**Comment by MagusWyvern** [comment]
Hello, wanted to contribute to this issue - Saya boleh tolong maintain fail terjemahan untuk Bahasa Melayu

**Comment by huseinzol05** [comment]
woah! sila sila! let me know if u need help yeeee

**Comment by zulkiflizaki** [comment]
> Hello, wanted to contribute to this issue - Saya boleh tolong maintain fail terjemahan untuk Bahasa Melayu

Ok, teruskan berusaha. Mohon maklumkan jika perlukan bantuan. Saya akan cuba membantu selagi mampu. Terima kasih dan semoga berjaya.

**Comment by MagusWyvern** [comment]
Saya akan cuba dengan forked punya repository dulu, first time guna readthedocs 

Nanti kalau ada apa apa saya komen 👍

**Comment by MagusWyvern** [comment]
I generated the translatable files as per readthedocs documentation [here](https://github.com/MagusWyvern/malaya-docs/tree/master/docs/locales/ms_MY/LC_MESSAGES), minta nasihat jika cara ini betul atau tidak.
